pom.xml

```diff
@@ -60,12 +48,6 @@
             <version>3.4</version>
```

Comment:
Could we push up the version to 4.0?

Reply:
The latest version appears to be 3.7? I have updated it to this.